src/features/auth/components/reset-password-form.tsx

```
@@ -0,0 +1,122 @@
+    resolver: zodResolver(resetPasswordFormSchema),
+    defaultValues: {
+      password: "",
+      confirmPassword: "",
+    },
+  });
+
+  const onSubmit = async (data: z.infer<typeof resetPasswordFormSchema>) => {
+    setIsLoading(true);
+    const result = await resetPasswordAction(data);
+
+    if (result && result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)